test/doc-views/TR/TRBase.js

```diff
@@ -63,6 +63,36 @@ export function buildCommonViewData(base) {
                     W3CLink: 'https://www.w.org/',
                 },
             },
+            niceCopyrightException: {
```

Comment:
I suggest renaming this section to `copyrightExceptionFound` to be consistent with the next one.
```suggestion
            copyrightExceptionFound: {
```